src/Symfony/Component/Validator/Resources/translations/validators.fi.xlf

```diff
@@ -366,6 +366,26 @@
                 <source>This value should be between {{ min }} and {{ max }}.</source>
                 <target>Arvon tulisi olla välillä {{ min }} - {{ max }}.</target>
             </trans-unit>
+            <trans-unit id="95">
+                <source>This value is not a valid hostname.</source>
+                <target>Arvo ei ole kelvollinen laitenimi (hostname).</target>
+            </trans-unit>
+            <trans-unit id="96">
+                <source>The number of elements in this collection should be a multiple of {{ compared_value }}.</source>
+                <target>Kokoelmassa olevien elementtien määrän pitää olla monikerta luvulle {{ compared_value }}.</target>
```

Comment:
Should we use "Ryhmässä" here instead on "Kokoelmassa"? Seems like "Ryhmä" term is used for that "Collection".

Reply:
Doh, this is exactly one of the cases where I did not agree with the existing vocabulary. Thought I changed to using "ryhmässä", but missed this one. You are of course correct, we should use the same wording everywhere.

Reply:
Personally I don't like that "ryhmä" term, but it's currently used everywhere so we should use that one - until we come up with a better alternative.